<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Temperature monitor</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QHBoxLayout" name="main_hlayout" stretch="0">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="left_vlayout">
        <item>
         <layout class="QHBoxLayout" name="broker_info_hlayout">
          <item>
           <widget class="QLabel" name="broker_lbl">
            <property name="text">
             <string>Broker</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="broker_hostname">
            <property name="toolTip">
             <string>Broker hostname</string>
            </property>
            <property name="text">
             <string>192.168.4.1</string>
            </property>
            <property name="placeholderText">
             <string>hostname</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="broker_port">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="toolTip">
             <string>Broker port (1-65535)</string>
            </property>
            <property name="maximum">
             <number>65535</number>
            </property>
            <property name="value">
             <number>1883</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="broker_connect_hlayout">
          <item>
           <widget class="QPushButton" name="clear_data_btn">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Clear data</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="broker_connect_btn">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="toolTip">
             <string>Connect to broker</string>
            </property>
            <property name="text">
             <string>Connect to broker</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QChartView" name="chart"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="temp_hlayout">
          <item>
           <layout class="QHBoxLayout" name="current_temp_hlayout">
            <item>
             <widget class="QLabel" name="curr_temp_lbl">
              <property name="text">
               <string>Current temperature:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="curr_temp_val_lbl">
              <property name="text">
               <string>-</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="curr_temp_degree_lbl">
              <property name="text">
               <string>°C</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="set_temp_lbl">
            <property name="text">
             <string>Temperature (°C)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="set_temp_input">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="minimum">
             <double>22.000000000000000</double>
            </property>
            <property name="maximum">
             <double>42.000000000000000</double>
            </property>
            <property name="value">
             <double>22.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="set_temp_btn">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Set</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="save_hlayout">
          <item>
           <layout class="QHBoxLayout" name="current_setpoint_hlayout">
            <item>
             <widget class="QLabel" name="curr_setpoint_lbl">
              <property name="text">
               <string>Current set temperature</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="curr_setpoint_val_lbl">
              <property name="text">
               <string>-</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="curr_setpoint_degree_lbl">
              <property name="text">
               <string>°C</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="save_data_btn">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Save data</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QtCharts</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
